src/config_builder.py

```diff
@@ -16,9 +16,12 @@
 
 LOKI_DIR = "/loki"
 CHUNKS_DIR = os.path.join(LOKI_DIR, "chunks")
+LOKI_CONFIG_PERSISTED = os.path.join(CHUNKS_DIR, "loki-local-config.yaml")
 COMPACTOR_DIR = os.path.join(LOKI_DIR, "compactor")
 BOLTDB_DIR = os.path.join(LOKI_DIR, "boltdb-shipper-active")
 BOLTDB_CACHE_DIR = os.path.join(LOKI_DIR, "boltdb-shipper-cache")
+TSDB_DIR = os.path.join(BOLTDB_DIR, "tsdb-index")
```

Comment:
As part of this PR, wdyt about adding something along the lines of the following comment to metadata.yaml?

```diff
    mounts:
+     # We do not need separate storages. TODO: In the next breaking change for Loki,
+     # switch to having just one persisted storate, e.g. `/loki/persisted`.
      - storage: active-index-directory
        location: /loki/boltdb-shipper-active
      - storage: loki-chunks
        location: /loki/chunks
```